File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        //高さを受け取る
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

//行
    for (int i = 0; i < height; i++)
    {
        //列
        for (int j = 0; j < height - (i + 1); j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        //谷
        printf("  ");
        for (int l = 0; l < i + 1; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int


# Prompt users type input
while True:
    height = get_int("Height: ")
    # Check whther the input is valid
    if 0 < height < 9:
        break

# Iterate for height times
for i in range(height):
    # Print left half of the pyramid
    for j in range(height - (i + 1)):
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")
    print("  ", end="")
    # Print right half of the pyramid
    for l in range(i + 1):
        print("#", end="")
    # Insert new line
    print()